events.md

```diff
@@ -1,9 +1,18 @@
 ---
 layout: page
 title: Events
-image: https://images.unsplash.com/photo-1535016120720-40c646be5580?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80
+description: The hub of all OLS events
+image: https://images.unsplash.com/photo-1503428593586-e225b39bddfe
 photos:
-  name: Alex Litvin
-  url: https://unsplash.com/photos/MAYsdoYpGuk
+  name: The Climate Reality Project
+  license: CC BY-SA 4.0
+  url: https://images.unsplash.com/photo-1503428593586-e225b39bddfe
 ---
 
+## Upcoming Events
+
+| Date/Time | Topic/Event | Venue/Location | Speakers/Contact | Organisers |
+|-----------|-------------|----------------|------------------|------------|
+{%- for event in site.data.events -%}
```

Comment:
You should compare the date here to have only the upcoming events and have another table below with past events

Reply:
Implemented. 👍 
This was a really good idea.